hack/local-up-karmada.sh

```diff
@@ -51,6 +51,19 @@ fi
 # get arch name and os name in bootstrap
 BS_ARCH=$(go env GOARCH)
 BS_OS=$(go env GOOS)
+BS_PATH=$(go env GOPATH)
+
+# install kwok
+kwok_version=v0.1.0
+echo -n "Preparing: 'kwokctl' existence check - "
+if util::cmd_exist kwokctl; then
+  echo "passed"
+else
+  echo "not pass, kwokctl is not found, and will install it..."
+  wget -O "${BS_PATH}/bin/kwokctl" "https://github.com/kubernetes-sigs/kwok/releases/download/${kwok_version}/kwokctl-${BS_OS}-${BS_ARCH}" && \
+    chmod +x "${BS_PATH}/bin/kwokctl"
+fi
```

Comment:
It seems `kwok` is only used in e2e test. Shall we move this check to e2e.sh? Most guys use this script just setup a karmada environment, `kwok` is not requisite.